types/react-select/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
When doing major update please do not edit the generated tslint and disable any rule.

Reply:
Thanks for the feedback @sheetalkamat 

I'll restore the original tslint.json and fix resulting linting issues.